app/src/main/java/com/soen490chrysalis/papilio/view/MainActivity.kt

```diff
@@ -1,21 +1,22 @@
 package com.soen490chrysalis.papilio.view
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.view.MenuItem
+import androidx.appcompat.app.AppCompatActivity
 import com.google.android.material.navigation.NavigationBarView
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.FirebaseUser
+import com.soen490chrysalis.papilio.*
 import com.soen490chrysalis.papilio.databinding.ActivityMainBinding
-import com.soen490chrysalis.papilio.ActivitiesFragment
-import com.soen490chrysalis.papilio.BrowseFragment
-import com.soen490chrysalis.papilio.HomeFragment
-import com.soen490chrysalis.papilio.UserProfileFragment
-import com.soen490chrysalis.papilio.R
 
 
 class MainActivity : AppCompatActivity(), NavigationBarView.OnItemSelectedListener
 {
     var bottomNavigationView: NavigationBarView? = null
     private lateinit var binding : ActivityMainBinding
+    private lateinit var firebaseAuth: FirebaseAuth
+    private lateinit var firebaseUser : FirebaseUser
```

Comment:
the firebase user and firebase auth variables can probably be removed since they are unused

Reply:
done